src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__block calendar__block--day--1"></div>
+      <div class="calendar__block calendar__block--day--2"></div>
```

Comment:
don't add additional clases
```suggestion
      <div class="calendar__block"></div>
      <div class="calendar__block"></div>
```